astropy/io/votable/tests/data/regression.bin.tabledata.truth.1.3.xml

```diff
@@ -258,7 +258,7 @@
      </TR>
     </TABLEDATA>
    </DATA>
-   <INFO ID="Error" name="Error" value="One might expect to find some INFO here, too..."/>
+   <INFO ID="ErrorInfo" name="Error" value="One might expect to find some INFO here, too..."/>
```

Comment:
This file is what we expect to get out when we load `regression.xml`, convert it to binary and back and output it in VOTable version 1.3.  In the original file, `regression.xml`, it used to be that only `name` was specified, so `ID` was created based on it.  In order to test that `ID` and `name` are also treated separately, I added an `ID` to the original that doesn't match `name`.